addSecurityExceptions.sh

```diff
@@ -45,10 +45,10 @@ if ! grep -q "networkSecurityConfig" "$tmpDir/AndroidManifest.xml"; then
 fi
 
 
-java -jar "$DIR/apktool.jar" empty-framework-dir --force "$tmpDir"
+java -jar "$DIR/apktool.jar" --use-aapt2 empty-framework-dir --force "$tmpDir"
 echo "Building temp APK $tempFileName"
-java -jar "$DIR/apktool.jar" b -o "./$tempFileName" "$tmpDir"
+java -jar "$DIR/apktool.jar" b --use-aapt2 -o "./$tempFileName" "$tmpDir"
 jarsigner -verbose -keystore $debugKeystore -storepass android -keypass android "./$tempFileName" androiddebugkey
-zipalign -p 4 $tempFileName $newFileName
+/Users/username/Library/Android/sdk/build-tools/30.0.3/zipalign -p 4 $tempFileName $newFileName
```

Comment:
This is great but it won't work 100% of the time. 
maybe there is a way to get the path programmatically?